include/eve/detail/spy.hpp

```diff
@@ -808,6 +814,11 @@ namespace avx512
 #   endif
 # endif
 #endif
+#if !defined(SPY_SIMD_DETECTED) && defined(__riscv) && defined(__riscv_vector)
+#  define SPY_SIMD_DETECTED ::spy::detail::simd_version::rvv_
+#  define SPY_SIMD_IS_RISCV_FLEXIBLE_SVE
```

Comment:
should it be just `SPY_SIMD_IS_RISCV_FLEXIBLE` ?
Also, care to make an issue/PR over at www.github.com/jfalcou/spy ?